control/init.sh

```diff
@@ -131,12 +131,24 @@ function ansible-deploy-control() {
     if [ -z "$1" ]; then
         echo "Usage:"
         echo "  ansible-deploy-control [environment]"
-        exit 1
+        return
     fi
     env="$1"
     echo "You must be root to deploy the control machine"
     echo "Run \`su\` to become the root user, then paste in this command to deploy:"
-    echo 'ENV='$env' && USER='`whoami` '&& ANSIBLE_DIR=/home/$USER/commcare-cloud/ansible && /home/$USER/.virtualenvs/ansible/bin/ansible-playbook -u root -i $ANSIBLE_DIR/inventories/localhost $ANSIBLE_DIR/deploy_control.yml -e @$ANSIBLE_DIR/vars/$ENV/${ENV}_vault.yml -e @$ANSIBLE_DIR/vars/$ENV/${ENV}_public.yml --diff --ask-vault-pass'
+    echo 'ENV=$env \
```

Comment:
Shouldn't this still be `'ENV='$env' \`, so that `$env` gets interpolated and printed out as "production" or whatever?